BridgeAppSDK/SBAUserWrapper+Bridge.swift

```diff
@@ -232,7 +232,7 @@ public extension SBAUserWrapper {
      @param completion  Completion handler
      */
     public func verifyRegistration(_ completion: ((Error?) -> Void)?) {
-        guard let username = self.email?(forAuthManager: nil), let password = self.password?(forAuthManager: nil) else {
+        guard let username = self.email?(forAuthManager: nil), let password = self.password(forAuthManager: nil) else {
```

Comment:
That's kind of confusing. Why would password be required but not email? Don't you need both to authenticate?